packages/calcite-components/src/components/rating/rating.e2e.ts

```diff
@@ -766,4 +770,59 @@ describe("calcite-rating", () => {
       expect(element).toEqualAttribute("value", "2");
     });
   });
+
+  describe("theme", () => {
+    describe("default", () => {
+      const tokens: ComponentTestTokens = {
+        "--calcite-rating-spacing": {
+          selector: `calcite-rating`,
+          targetProp: "--calcite-internal-rating-spacing",
+        },
+        "--calcite-rating-outline-color": {
+          shadowSelector: `.${CSS.star}`,
+          targetProp: "color",
+        },
+        "--calcite-rating-outline-color-hover": {
+          shadowSelector: "label",
+          targetProp: "color",
+          state: "hover",
+        },
+        "--calcite-rating-value-fill-color": {
+          shadowSelector: "label",
+          targetProp: "color",
+          state: { press: { attribute: "class", value: "star" } },
+        },
+      };
+      themed(async () => html`<calcite-rating></calcite-rating>`, tokens);
+    });
+    describe("average with chip", () => {
+      const tokens: ComponentTestTokens = {
+        "--calcite-rating-average-fill-color": {
+          shadowSelector: `.${CSS.average}`,
+          targetProp: "color",
+        },
+        "--calcite-rating-chip-background-color": {
+          shadowSelector: "calcite-chip",
+          targetProp: "--calcite-chip-background-color",
+        },
+        "--calcite-rating-chip-border-color": {
+          shadowSelector: "calcite-chip",
+          targetProp: "--calcite-chip-border-color",
+        },
+        "--calcite-rating-chip-shadow": {
+          shadowSelector: "calcite-chip",
+          targetProp: "--calcite-chip-shadow",
+        },
+        "--calcite-rating-chip-corner-radius": {
+          shadowSelector: "calcite-chip",
+          targetProp: "--calcite-chip-corner-radius",
+        },
+        "--calcite-rating-chip-text-color": {
+          shadowSelector: "calcite-chip",
+          targetProp: "--calcite-chip-text-color",
+        },
+      };
+      themed(async () => html`<calcite-rating average="3.65" show-chip></calcite-rating>`, tokens);
```

Comment:
You don't need the async function if the markup is ready for testing, so this can be simplified to

```ts
themed(html`<calcite-rating average="3.65" show-chip></calcite-rating>`, tokens);
```